x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -819,6 +819,9 @@ export class Embeddable
   }
 
   public canViewUnderlyingData() {
+    if (!this.savedVis) {
```

Comment:
Why is it needed here? 🤔 
The `loadViewUnderlyingDataArgs` is already checking this. 

```
if (!this.activeDataInfo.activeData || !mergedSearchContext.timeRange || !this.savedVis) {
      return false;
    }
```

Reply:
I agree—that's better.